### PR TITLE
Add button to get a link to the event

### PR DESCRIPTION
**Pull request by anttitikkanen**
+ What existing problem does this PR solve?

https://github.com/google/timesketch/issues/2270 (but just for the legacy UI, will send a separate PR for the new one)

+ What new feature is being introduced with this PR?

Copy a link to an individual event, so it's easier to refer to it from external sources.

+ Overview of changes to existing functions if required.

**Checks**

- [x ] All tests succeed.
- [ ] Unit tests added.
- [ ] e2e tests added.
- [ ] Documentation updated.

**Closing issues**

### CODE REVIEW

File: timesketch/frontend/src/components/Explore/EventListRowDetail.vue [hunk 1]
```diff
@@ -157,6 +167,9 @@ export default {
       }
       return regexes
     },
+    getEventLink: function () {
+      return encodeURI(`${window.location.origin}/sketch/${this.$store.state.sketch.id}/explore?q=_id:"${this.event._id}"&timeline=${this.event._source.__ts_timeline_id}`);
+    },
```
Comment (tomchop) [inline timesketch/frontend/src/components/Explore/EventListRowDetail.vue:172]: It would be preferable to leverage the Vue router to generate this link

**Review by tomchop — CHANGES_REQUESTED** [review]

File: timesketch/frontend/src/components/Explore/EventListRowDetail.vue (hunk 1 above)
Comment (jaegeral) [reply timesketch/frontend/src/components/Explore/EventListRowDetail.vue:172]: @anttitikkanen would you mind to make that with Vue? then this should be good to be merged.

**Comment by jaegeral** [comment]
@anttitikkanen friendly ping

**Comment by berggren** [comment]
Closing this one out in lack of response. The old UI is to be deprecated in the near future. Tracking this feature in the new UI here: #2371

**Pull request closed without merging**